src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
+    this.info = data
+    this.priority = priority
+    this.before = proceeds
+  }
+}
+
+class priorityQueue {
+  constructor() {
+    this.head = null
+    this.tail = null
+    this.count = 0
+  }
+}
+
+  enqueue(value, num) {// adds an element with priority (number) to the back of the queue.
```

Comment:
As a matter of code style, I prefer placing comments *before* the line of code they reference, so that when I am reading the code from top to bottom, I read the relevant comment before the line of code.  As well, this comment  may be longer than some folks' editor window width, and scroll of the screen - If it's important, we want to make it easy for other developers to read it.